pkgs/servers/clickhouse/default.nix

```diff
@@ -206,7 +212,7 @@ in mkDerivation rec {
     homepage = "https://clickhouse.com";
     description = "Column-oriented database management system";
     license = licenses.asl20;
-    maintainers = with maintainers; [ orivej ];
+    maintainers = with maintainers; [ orivej mbalatsko ];
```

Comment:
@mbalatsko IIRC you need to be in a list of maintainers elsewhere in nixpkgs to add yourself here. If you are I'm fine with this change but if not, you may get an evaluation error.

Reply:
I am a maintainer already ;)